### PR TITLE
Локализация - guide/built-ins/teleport.md

### PR DESCRIPTION
**Pull request by lordoys**
resolves https://github.com/translation-gang/docs-ru/issues/142

### CODE REVIEW

**Comment by lordoys** [comment]
@Alex-Sokolov

File: src/guide/built-ins/teleport.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
-# Teleport {#teleport}
+# Телепорт {#teleport}
 
  <VueSchoolLink href="https://vueschool.io/lessons/vue-3-teleport" title="Free Vue.js Teleport Lesson"/>
```
Comment (Alex-Sokolov) [inline src/guide/built-ins/teleport.md:3]:
```suggestion
 <VueSchoolLink href="https://vueschool.io/lessons/vue-3-teleport" title="Бесплатный урок о телепортах во Vue.js"/>
```

File: src/guide/built-ins/teleport.md [hunk 2]
```diff
@@ -146,57 +146,57 @@ let open = $ref(false)
...
+  <div>Б</div>
 </div>
 ```
 
 ---
 
-**Related**
+**Связанные**
 
 - [`<Teleport>` API reference](/api/built-in-components.html#teleport)
```
Comment (Alex-Sokolov) [inline src/guide/built-ins/teleport.md:201]:
```suggestion
- [Справочник API — `<Teleport>`](/api/built-in-components.html#teleport)
```

File: src/guide/built-ins/teleport.md [hunk 3]
```diff
@@ -146,57 +146,57 @@ let open = $ref(false)
...
 - [Handling Teleports in SSR](/guide/scaling-up/ssr.html#teleports)
```
Comment (Alex-Sokolov) [inline src/guide/built-ins/teleport.md:202]:
```suggestion
- [Работа с телепортами в SSR](/guide/scaling-up/ssr.html#teleports)
```

**Review by Alex-Sokolov — APPROVED** [review]

**Comment by Alex-Sokolov** [comment]
Спасибо!

**Pull request merged**